modules/geo-layers/src/h3-layers/h3-hexagon-layer.ts

```diff
@@ -266,12 +335,13 @@ export default class H3HexagonLayer extends CompositeLayer {
     };
   }
 
-  _renderPolygonLayer() {
+  _renderPolygonLayer(): PolygonLayer {
     const {data, getHexagon, updateTriggers, coverage} = this.props;
 
     const SubLayerClass = this.getSubLayerClass('hexagon-cell-hifi', PolygonLayer);
     const forwardProps = this._getForwardProps();
 
+    // @ts-ignore
```

Comment:
Comment what is being ignored

Reply:
Fixed by adding a type to `updateTriggers`